sample/src/main/java/kin/sdk/core/sample/ViewUtils.java

```diff
@@ -28,6 +33,32 @@ public static void alert(Context context, String message) {
         dialog.show();
     }
 
+    public static void confirmAlert(Context context, String message, String button, onAlertClicked onAlertClicked) {
```

Comment:
A suggestion: create BaseDialog that extends from Dialog and receive in the constructor title, message, confirmListener.
and create DeleteDialog that extends BaseDialog.